src/modules/twap/pure/PrimaryActionButton/getTwapFormState.tsx

```diff
@@ -28,5 +37,9 @@ export function getTwapFormState(props: TwapFormStateParams): TwapFormState | nu
     return TwapFormState.NEED_FALLBACK_HANDLER
   }
 
+  if (isSellAmountTooSmall(sellAmountPartFiat, chainId)) {
```

Comment:
Also, loading of fiat prices takes some time. I think we shouldn't allow placing an order while a fiat price is not loading. Maybe we should return `TwapFormState.LOADING_SAFE_INFO`. It's not a blocker now, you can just leave a TODO

Reply:
I don't think that should be a blocker because it might be down.
At least this is quite common with Goerli.
Yes, testnet, but we for instance don't block trades on mainnet when USD amounts are missing